module-services/service-cellular/src/CSQHandler.cpp

```diff
@@ -8,70 +8,78 @@
 #include <ticks.hpp>
 #include <chrono>
 
-namespace cellular::service
+namespace
 {
+    constexpr auto urcThreshold     = 4;
+    constexpr auto pollModeDuration = std::chrono::minutes{60};
+
+    bool isRssiValid(std::uint32_t csq)
+    {
+        constexpr auto invalidRssiLow  = 99;
+        constexpr auto invalidRssiHigh = 199;
+        return ((csq != invalidRssiLow) && (csq != invalidRssiHigh));
```

Comment:
csq can't has these 2 particular value, and all other are ok?
or should it be more like this:
return ((csq > invalidRssiLow) && (csq < invalidRssiHigh))
?

Reply:
TBH I have no idea - I just copied this code from another place and put in separate function to improve readability